Quadriga/src/main/webapp/WEB-INF/views/workbench/addpublicpage.jsp

```diff
@@ -0,0 +1,151 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<%@taglib uri="http://www.springframework.org/tags/form" prefix="form"%>
+
+<!-- Content -->
+<link rel="stylesheet"
+	href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
+<script>
+	function performAction(idValue) {
+		var data = {};
+		var titles = $("#title" + idValue).val();
```

Comment:
@jdamerow  yes i can do this. But wouldn't it require me to write three methods for each of the buttons?